bcipy/signal/model/pca_rda_kde/pca_rda_kde.py

```diff
@@ -0,0 +1,126 @@
+import pickle
+from pathlib import Path
+from typing import List
+
+import numpy as np
+from bcipy.signal.model.base_model import ModelEvaluationReport, SignalModel
+from bcipy.signal.model.pca_rda_kde.classifier import RegularizedDiscriminantAnalysis
+from bcipy.signal.model.pca_rda_kde.cross_validation import cost_cross_validation_auc, cross_validation
+from bcipy.signal.model.pca_rda_kde.density_estimation import KernelDensityEstimate
+from bcipy.signal.model.pca_rda_kde.dimensionality_reduction import ChannelWisePrincipalComponentAnalysis
+from bcipy.signal.model.pca_rda_kde.pipeline import Pipeline
+from bcipy.signal.exceptions import SignalException
+
+
+class PcaRdaKdeModel(SignalModel):
+    def __init__(self, k_folds=10):
```

Comment:
We chatted briefly, but I'm in favor of removing these defaults. It can prevent a lot of headaches down the line.